### DISCUSSION TITLE
Is it possible for non-rooted device in the future?

### DISCUSSION THREAD

**Discussion by REDDDDY** [Mygod/VPNHotspot#382] [General]
As per title, would this app be possible to be installed and working on Non-Rooted device in the future release?
I hope devs can help make this a reality 😎🙂

I have already gone through the whole discussion to see if this was originally asked or not before posting 🙂

**Comment by Mygod** [comment]
No plan for this. See also https://github.com/Mygod/VPNHotspot/issues/62.
